src/java.base/share/classes/java/lang/System.java

```diff
@@ -124,10 +124,19 @@ private System() {
      *
      * @apiNote
      * The typical approach to read character data is to wrap {@code System.in}
-     * within an {@link java.io.InputStreamReader InputStreamReader} or other object
-     * that handles character encoding. After this is done, subsequent reading should
-     * use only the wrapper object; operating directly on {@code System.in} results
-     * in unspecified behavior.
+     * within the object that handles character encoding. After this is done,
+     * subsequent reading should use only the wrapper object; operating directly
+     * on {@code System.in} results in unspecified behavior.
+     * <p>
+     * Here are two common examples. Using an {@link java.io.InputStreamReader
+     * InputStreamReader}:
+     * {@snippet lang=java :
+     *     new InputStreamReader(System.in, System.getProperty("stdin.encoding"));
+     * }
+     * Or with a {@link java.util.Scanner Scanner}:
```

Comment:
"Or with" -> "Using a" or "Or using a" for consistency? Up to you.

Reply:
Thanks. That flows better. Corrected.

Reply:
You're welcome. Re-approved.